packages/server/src/api/controllers/static/index.ts

```diff
@@ -167,10 +222,23 @@ export const serveApp = async function (ctx: Ctx) {
     const appInfo = await db.get<any>(DocumentType.APP_METADATA)
     let appId = context.getAppId()
 
+    const hideDevTools = !!ctx.params.appUrl
+    const sideNav = appInfo.navigation.navigation === "Left"
```

Comment:
Where is this `Left` magic string coming from? Don't we have any type we can use?

Reply:
It's the positioning of the navigation. I don't believe there's currently any type or constant for it, both the place it's set and the place it's consumed currently use the raw string:

https://github.com/Budibase/budibase/blob/app-skeleton-2/packages/builder/src/pages/builder/app/%5Bapplication%5D/design/%5BscreenId%5D/%5BcomponentId%5D/_components/Navigation/index.svelte#L90
https://github.com/Budibase/budibase/blob/app-skeleton-2/packages/client/src/components/app/Layout.svelte#L41